lms/envs/common.py

```diff
@@ -212,9 +212,9 @@
     # when enrollment exceeds this number
     'MAX_ENROLLMENT_INSTR_BUTTONS': 200,
 
-    # Grade calculation started from the new instructor dashboard will write
-    # grades CSV files to S3 and give links for downloads.
-    'ENABLE_S3_GRADE_DOWNLOADS': False,
+    # Grade calculation started from the instructor dashboard will write grades
+    # CSV files to the configured storage backend and give links for downloads.
+    'ENABLE_GRADE_DOWNLOADS': False,
```

Comment:
I agree with that approach. We can add a note / checkbox in the wiki page for the F* release to make sure we clean it up - add it as a prereq or something. We should create that page if we haven't already.